src/iuse_actor.cpp

```diff
@@ -1986,6 +1988,11 @@ bool could_repair( const player &p, const item &it, bool print_msg )
     return true;
 }
 
+bool repair_item_actor::can_use( const player*, const item* it, bool, const tripoint& ) const
+{
+    return it->charges >= charges_to_use;
```

Comment:
This is inconsistence: it only checks the required charges from the actor, but not the required charges from `it_tool`. It does not match the changed logic in activity_handlers.cpp, mentioned above.